werk24/models/bom_table.py

```diff
@@ -0,0 +1,76 @@
+from typing import List, Optional
+
+from pydantic import BaseModel
+
+from werk24.models.material import W24MaterialOption
+from werk24.models.weight import W24Weight
+from werk24.models.value import W24PhysicalQuantity
+
+
+class W24BomQuantity(BaseModel):
+    """Quantity of parts listed in BOM
+
+    Args:
+        quantity (str): Quantity of the part. 
+        unit (Optional[str]): Unit of the quantity. 
+    """
+    quantity: str
```

Comment:
W24BomQuantity should be removed - as discussed

Reply:
W24PhysicalQuantity is the correct class for this

Reply:
removed